File: unit-3/sprint-2/evaluation/firstapp/app/src/main/java/com/example/firstapp/ResponseModel.java
package com.example.firstapp;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class ResponseModel implements Serializable {

	@SerializedName("image")
	private String image;

	@SerializedName("subTitle")
	private String subTitle;

	@SerializedName("title")
	private String title;

	public String getImage(){
		return image;
	}

	public String getSubTitle(){
		return subTitle;
	}

	public String getTitle(){
		return title;
	}

	@Override
 	public String toString(){
		return 
			"ResponseModel{" + 
			"image = '" + image + '\'' + 
			",subTitle = '" + subTitle + '\'' + 
			",title = '" + title + '\'' + 
			"}";
		}
}

File: unit-3/sprint-4/day-2/you/18AugPostRequest/app/src/main/java/com/example/a18augpostrequest_kotlin/ResponseModel.kt
package com.example.a18augpostrequest_kotlin

import com.google.gson.annotations.SerializedName
import java.io.Serializable


class ResponseModel : Serializable {
    @SerializedName("email")
    val email: String? = null

    @SerializedName("password")
    val password: String? = null

    @SerializedName("title")
    val title: String? = null

    @SerializedName("requestType")
    val requestType: String? = null

    @SerializedName("id")
    val id = 0
}